### ISSUE TITLE
Migrate current iris database to new iris cluster

### ISSUE THREAD

**Issue by steveellis** [culibraries/folio#146]
@jwfuller seems like we needed a ticket for this. There are a lot of different ways to do this. Since we're using Aurora we might want to take advantage of some features of it in particular for this kind of thing.

**Comment by steveellis** [comment]
It looks like aurora cloning is what we want: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Managing.Clone.html. Has lots of nice features, like quick clones for testing environments. Even if we can't do it now, we should likely use it in the future.

But I'm concerned that it won't work for what we have. I've got 3 availability zones for the new cluster, but I don't believe our current aurora cluster has 3 (need to verify this). For cloning to work we need for the source and the clone to share the same AZs. This is a limitation of cloning. From the doc:

> You can create a clone in a different virtual private cloud (VPC) than that of the Aurora DB cluster. If you do, the subnets of the VPCs must map to the same Availability Zones.

I created 3 AZs for the RDS cluster because anything less seemed to break certain things in pulumi and the docs stated that 3 are recommended.
